_plugins/convert_md.rb

```diff
@@ -1,9 +1,9 @@
 # Convert the links that point to a 'README.md' to 'index.html'
 Jekyll::Hooks.register :pages, :post_render do |doc|
-  doc.output = doc.output.gsub(/(href="\.?\.\/[^.]*)README\.md?(#(.*))?"/, '\1index.html\2"')
+  doc.output = doc.output.gsub(/(href="\.?\.?\/?\.?\.?\/?\.?\.\/[^.]*)README\.md?(#(.*))?"/, '\1index.html\2"')
```

Comment:
Is there not a more elegant way to express this? :smile: Something like:

```suggestion
  doc.output = doc.output.gsub(/(href="((\.\.)/?)+\/[^.]*)README\.md?(#(.*))?"/, '\1index.html\2"')
```

Not a problem if not.